src/updateTime/updateTimeReleaser.ts

```diff
@@ -5,18 +5,23 @@ import { TasksClient } from '../clients/tasksClient';
 import { Services } from '../common/constants';
 import { IConfig } from '../common/interfaces';
 import { toBoolean } from '../common/utilities/typeConvertors';
+import { HeartbeatClient } from '../clients/heartbeatClient';
+import { NotFoundError } from '../common/exceptions/http/notFoundError';
 
 @injectable()
 export class UpdateTimeReleaser {
   private readonly enabled: boolean;
+  private readonly checkHeartbeatEnabled: boolean;
 
   public constructor(
     @inject(Services.CONFIG) config: IConfig,
     @inject(Services.LOGGER) private readonly logger: Logger,
     @inject(Services.TRACER) private readonly tracer: Tracer,
-    private readonly tasksClient: TasksClient
+    private readonly tasksClient: TasksClient,
+    private readonly heartbeatClient: HeartbeatClient
   ) {
     this.enabled = toBoolean(config.get('updateTime.enabled'));
+    this.checkHeartbeatEnabled = toBoolean(config.get('updateTime.checkHeartbeat'));
```

Comment:
```suggestion
    this.checkHeartbeatEnabled = config.get<boolean>('updateTime.checkHeartbeat');
```

Reply:
fixed and deleted common/utilities/typeConvertors.ts that contained toBoolean function that is no longer in use